src/collections/events/kubecon-na-2022/index.mdx

```diff
@@ -34,7 +34,7 @@ Join <Link to="/blog/events/layer5-at-kubecon-cloudnativecon-north-america">Laye
 ### Service Mesh Performance Project Office Hours
   
 <p>
-  Service Mesh Performance is a CNCF-hosted project that has received support from Layer5, Intel, RedHat, and HashiCorp. By leveraging SMP, MeshMark provides a global performance index that allows you to assess how well your mesh deployment stacks up against those of other businesses. 
+  Service Mesh Performance is a CNCF-hosted project that has received support from Layer5, Intel, RedHat, and HashiCorp. Join the Maintainer, Service Mesh Performance and the cloud software developer from Intel <Link to="/community/members/xin-huang">Xin Huang</Link> with the Founder of Layer5 <Link to="/community/members/lee-calcote">Lee Calcote</Link> as they discuss SMP project in detail. By leveraging SMP, MeshMark provides a global performance index that allows you to assess how well your mesh deployment stacks up against those of other businesses. 
```

Comment:
Uh-oh. This is missing two commas:

"...Founder of Layer5, Lee Calcote, as they..."